src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/ReissueFurtherEvidenceHandler.java

```diff
@@ -121,21 +113,6 @@ private String getNoSelectedDocumentErrorMessage(SscsCaseData caseData) {
             caseData.getCcdCaseId());
     }
 
-    private void udateCaseForReasonableAdjustments(SscsCaseData caseData, AbstractDocument selectedDocument) {
-        if (caseData.getReasonableAdjustmentsLetters() != null) {
```

Comment:
you have effectively removed this condition - what effect does that have on the caseData in the V2 call?